debian/changelog

```diff
@@ -1,3 +1,14 @@
+libfm-qt (1.1.0-0ubuntu7~ppa2) jammy; urgency=medium
+
+  * Backport to Jammy.
+  * Updated Lubuntu Developers copyright.
+  * Updated symbols file
+  * Removed Repository-Browser field from debian/upstream/metadata.
+  * Removed "-Wl,--as-needed" linker flag.
+  * Bumped Standards-Version (no changes necessary).
```

Comment:
Technically it's `Bumped Standards-version to 4.6.1, [no changes needed / (no chances necessary).` Sorry for being anal. :hugs:

Reply:
Fixed.